specification_documents/mzTab_format_specification_2_1-M.adoc

```diff
@@ -906,6 +906,21 @@ MTD ms_run[2]-id_format [MS, MS:1000774, multiple peak list nativeID format, ]
 ....
 |==================================================================================================================================================================
 
+[[ms_run1-n-usi_identifier]]
+==== ms_run[1-n]-usi_identifier
+[cols=",",]
+|===================================================================================================================================================================
+|*Description* |An identifier for the MS run based on the Universal Spectrum Identifier (USI) specification.
+Implied within the USI is an MS run identifier. Every deposited MS run can be referenced with a shortened form: `mzspec:<collection>:<msRun>`.
+Since an MS run may be represented in several formats (with potentially slightly different data associated with the spectrum), a format suffix MAY be specified (e.g. `mzspec:<collection>:<msRun>.RAW`) to signify that a specific file type is meant.
```

Comment:
If specified, the format suffix should align with the `ms_run[1-n]-format` parameter and the `ms_run[1-n]-location` suffix defined in the metadata section.